test/e2e/app-dir/parallel-routes-and-interception/parallel-routes-and-interception.test.ts

```diff
@@ -187,38 +186,6 @@ createNextDescribe(
         expect(pageText).toContain('parallel/(new)/@baz/nested/page')
       })
 
-      it('should throw an error when a route groups causes a conflict with a parallel segment', async () => {
```

Comment:
this test was reworked and moved [here](https://github.com/vercel/next.js/pull/59791/files#diff-0d5e642ee398c97861073a49292d556fed32e72bd738cdc793509572e89409b3)